msticpy/data/drivers/odata_driver.py

```diff
@@ -229,6 +227,21 @@ def query_with_results(  # noqa: MC0001
 
     # pylint: enable=too-many-branches
 
+    @staticmethod
+    def _check_response_errors(response):
+        """Check the response for possible errors."""
+        if response.status_code == requests.codes["ok"]:
```

Comment:
I was not aware of requests.codes thats cool!

Reply:
Weird! I thought that was your code!